content/en/account_management/saml/activedirectory.md

```diff
@@ -11,104 +11,25 @@ further_reading:
-
-{{< img src="account_management/saml/xex71aV.png" alt="xex71aV" style="width:60%;">}}
-
-Finish by clicking Close. This saves the trust definition and open the claims window where you may add a couple of recommended Claim Rules.
-
-{{< img src="account_management/saml/5NkUanW.png" alt="5NkUanW" style="width:60%;">}}
-
-Datadog recommends two Claim Rules for brokering the SAML assertions. They can be added by first clicking the Add Rule button.
-
-{{< img src="account_management/saml/QkNaDCD.png" alt="QkNaDCD" style="width:60%;">}}
-
-This first rule is an LDAP Attributes rule that ensures the required information is passed between the two systems. Configure the rule as shown below and click OK to save. (Make sure to use three separate fields for "E-Mail-Addresses, Given-Name, and Surname" or else some relevant info may be left as "None" later on.)
-
-{{< img src="account_management/saml/cogaUQT.png" alt="cogaUQT" style="width:60%;">}}
-
-The second rule is a Transform rule. Datadog specifies `urn:oasis:names:tc:SAML:1.1:nameid-format:emailAddress` for the format of the NameIDPolicy in assertion requests where ADFS natively expects these in Name ID format so you need to transform the format from email to Name ID.
-
-Select Transform an Incoming Claim from the drop-down and click **Next** to continue.
-
-{{< img src="account_management/saml/JS5FNbR.png" alt="JS5FNbR" style="width:60%;">}}
-
-Input the configuration as shown below and click **Finish**.
-
-{{< img src="account_management/saml/OT9i0K5.png" alt="OT9i0K5" style="width:60%;">}}
-
-Save the new claim rules by clicking OK.
-
-{{< img src="account_management/saml/CeCyDmc.png" alt="CeCyDmc" style="width:60%;">}}
-
-Download and import the ADFS identity provider metadata from the ADFS server into the SAML configuration in your [Datadog Organization Saml page][4]
-
-This file can be downloaded from the following URL (replace hostname with the public DNS hostname of your server) - `https://hostname/FederationMetadata/2007-06/FederationMetadata.xml`
-
-Import into your Datadog Organization from the SAML configuration page as shown below:
-
-{{< img src="account_management/saml/KJxaVYe.png" alt="KJxaVYe" style="width:60%;">}}
-
-That's it! Once SAML is configured, users can login by using the link provided in the SAML configuration page.
-
-Keep in mind that users still need to be invited and activated before they're able to login. Be sure to invite new users by using the email address corresponding to their Active Directory user record otherwise they may be denied as shown below.
+Once SAML is configured, users can login by using the link provided in the [SAML configuration page][3]. Keep in mind that users still need to be invited and activated before they're able to login. Be sure to invite new users by using the email address corresponding to their Active Directory user record otherwise they may be denied as shown below.
```

Comment:
```suggestion
Once SAML is configured, users can login by using the link provided in the [SAML configuration page][3]. Keep in mind that users still need to be invited and activated before they're able to login. Be sure to invite new users by using the email address corresponding to their Active Directory user records; otherwise they may be denied as shown below.
```